boot-microservices/src/main/java/org/ow2/proactive/boot/microservices/iam/util/IAMJsonUtils.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * ProActive Parallel Suite(TM):
+ * The Open Source library for parallel and distributed
+ * Workflows & Scheduling, Orchestration, Cloud Automation
+ * and Big Data Analysis on Enterprise Grids & Clouds.
+ *
+ * Copyright (c) 2007 - 2017 ActiveEon
+ * Contact: contact@activeeon.com
+ *
+ * This library is free software: you can redistribute it and/or
+ * modify it under the terms of the GNU Affero General Public License
+ * as published by the Free Software Foundation: version 3 of
+ * the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ * If needed, contact us to obtain a release under GPL Version 2 or 3
+ * or a different license than the AGPL.
+ */
+package org.ow2.proactive.boot.microservices.iam.util;
+
+import java.util.Map;
+
+import org.jose4j.json.internal.json_simple.JSONObject;
+import org.jose4j.json.internal.json_simple.parser.JSONParser;
+import org.jose4j.json.internal.json_simple.parser.ParseException;
+
+
+public class IAMJsonUtils {
+
+    /**
+     * Attributes used to parse a (json) credentials validation answer from IAM
+     * Example of json answer:
```

Comment:
Maybe you can put a version number for this example.

Reply:
I prefer not adding a version for this, to avoid updating it (in the scheduling project) each time the iam project is updated. If the json produced by IAM changes, this will create issues in the scheduling project. The code and related javadoc will be then updated, accordingly.